README.md

```diff
@@ -17,25 +17,39 @@ Following platform is required to run the API Mediation Layer:
 
 Following tools are required to build and develop API Mediation Layer:
 
-Nodejs and npm are required to be installed globally to be able to build the API Catalog ui
+* Node.js and npm are required to be installed globally to be able to build the API Catalog UI
 
 ## Quick start
 
 1. Install the package manager `pnpm` globally in order to build the project:
-    
-        npm add -g pnpm
-    
-2. Install `concurrently` globally:
-   
-       npm install -g concurrently
-       
-3. Build all modules:
 
-       ./gradlew build
+    ```sh
+    npm add -g pnpm
+    ```
 
-4. Run all service on local machine:
+2. Install npm packages for the UI:
 
-       npm run api-layer
+    ```sh
+    cd api-catalog-ui/frontend/; pnpm install; cd ../..
+    ```
+
```

Comment:
I had to run it for the first build and then every time after `./gradlew clean` otherwise the build has failed. It took some time to figure this out so I had added this step so new users are not struglling.